back-nest/src/sсrapers/models/scraped-product.model.ts

```diff
@@ -0,0 +1,11 @@
+export interface ScrapedProduct {
+  title: string;
+  subtitle: string;
+  description: string;
+  price: number;
+  specifications: string;
+  type: string;
+  profileImage: string | null;
+  source: string;
+  newPrice?: number;
```

Comment:
in your mappers return null if `newPrice` doesnt exist

```suggestion
  newPrice: number | null;
```